code/modules/mob/living/simple_animal/abnormality/he/scaredy_cat.dm

```diff
@@ -0,0 +1,205 @@
+	return ..()
+
+//if he still has a friend, revives
+/mob/living/simple_animal/hostile/abnormality/scaredy_cat/proc/Regenerate()
+	anchored = FALSE //so it can't be knocked away from his stunned effect
+	if(friend)
+		stunned_effect = null
+		density = TRUE
+		revive(full_heal = TRUE, admin_revive = TRUE)
+		GoToFriend()
+		return
+	animate(src, alpha = 0, time = 3 SECONDS)
+	QDEL_IN(src, 3 SECONDS)
+
+///teleports to his friend if he can't see them
+/mob/living/simple_animal/hostile/abnormality/scaredy_cat/proc/GoToFriend()
+	if(!friend)
+		return
+	var/turf/origin = get_turf(friend)
+	var/list/all_turfs = RANGE_TURFS(2, origin)
+	for(var/turf/T in all_turfs)
+		if(T.density || T == origin || !can_see(friend, T, 2))
+			continue
+		playsound(src, 'sound/abnormalities/scaredycat/cateleport.ogg', 50, FALSE, 4)
+		forceMove(T)
+		for(var/mob/living/carbon/human/enemy in oview(src, vision_range))
+			if(enemy.stat != DEAD)
+				GiveTarget(enemy) //the moment he teleports he's already on the offensive
+				break
+		return
+
+///If scaredy cat becomes a big boy or a baby boy
+/mob/living/simple_animal/hostile/abnormality/scaredy_cat/proc/Courage(courage)
+	if(courage)
+		melee_damage_lower = 25
+		melee_damage_upper = 30
+		damage_coeff = list(BRUTE = 1, RED_DAMAGE = 0.5, WHITE_DAMAGE = 2, BLACK_DAMAGE = 1.5, PALE_DAMAGE = 0.5)
+		icon = 'ModularTegustation/Teguicons/48x48.dmi'
+		icon_living = "cat_courage"
+		icon_dead = "dead_courage"
+	else
+		faction = list("neutral")
+		melee_damage_lower = initial(melee_damage_lower)
+		melee_damage_upper = initial(melee_damage_upper) //it shouldn't attack in that form in the first place but...
+		damage_coeff = list(BRUTE = 1, RED_DAMAGE = 4, WHITE_DAMAGE = 4, BLACK_DAMAGE = 4, PALE_DAMAGE = 4)
+		playsound(src, 'sound/abnormalities/scaredycat/catchange.ogg', 75, FALSE, 4)
+		icon = 'ModularTegustation/Teguicons/32x32.dmi'
+		icon_living = "scaredy_cat"
+		icon_dead = "scaredy_dead"
+		if(stunned_effect)
+			animate(stunned_effect, alpha = 0, time = 1 SECONDS)
+			QDEL_IN(stunned_effect, 1 SECONDS)
+			stunned_effect = null
+	if(stat != DEAD) //because update_icon() won't do its job properly
+		icon_state = icon_living
+	else
+		icon_state = icon_dead
+
+/mob/living/simple_animal/hostile/abnormality/scaredy_cat/proc/OnMobDeath(datum/source, mob/living/died, gibbed)
+	SIGNAL_HANDLER
+	if(status_flags & GODMODE)
+		if(istype(died, /mob/living/simple_animal/hostile/abnormality))
+			datum_reference.qliphoth_change(1) //counter increase if another abno dies
+		return FALSE
+	if(died == friend)
+		friend = null
+		Courage(FALSE)
+		if(stat != DEAD)
+			wait_for_friend = TRUE //kill him fast before he finds another friend
+		return TRUE
+	return FALSE
+
+/mob/living/simple_animal/hostile/abnormality/scaredy_cat/proc/OnAbnoBreach(datum/source, mob/living/simple_animal/hostile/abnormality/abno)
+	SIGNAL_HANDLER
+	if(abno.name == "Standard training-dummy rabbit")
```

Comment:
Make this if the abnormality is a training one, might breach into the tutorial area.